lucien-ios/AddAndEditNewComicBook/ComicFormViewModel.swift

```diff
@@ -16,17 +19,23 @@ enum  ComicFormMode {
 struct ComicFormViewModel {
     var coverPhoto: UIImage?
     var comicFormMode: ComicFormMode
-    var seriesTitle: String?
-    var volume: String?
-    var storyTitle: String?
-    var issue: String?
-    var publisher: String?
-    var release: String?
+    var seriesTitle = Variable<String>("")
```

Comment:
If you initialize this with an initial value (the '("")' part, you don't need to explicitly declare it's a variable of Strings. Also this can be a let. Aka you can just say this `let seriesTitle = Variable("")`. Same for the ones below